uiowa/uiowa
# Added potential Google Analytics 4 fix

Closes #2614 


# How to test
Run composer install; composer update drupal/google_analytics; install site and see changes made there. Not sure how to test actual analytics without a life site to reference
